dev/TeachingTip/TeachingTip.cpp

```diff
@@ -971,27 +975,58 @@ void TeachingTip::OnF6AcceleratorKeyClicked(const winrt::CoreDispatcher&, const
             m_hasF6BeenInvoked = true;
         }
 
-        winrt::Button f6Button = [this]() -> winrt::Button
+        auto const focusFromContent = [this, args]()
         {
-            auto firstButton = m_closeButton.get();
-            auto secondButton = m_alternateCloseButton.get();
-            if (CloseButtonContent())
+            auto parent = winrt::FocusManager::GetFocusedElement();
```

Comment:
>parent [](start = 17, length = 6)

This could be a Hyperlink or a ContentLink, neither of which is a UIElement, but both are focusable.  The loop below will fail to cast this to UIElement, and fail to follow the parent chain.

I believe VisualTreeHelper.GetParent will work for both Hyperlink and ContentLink, so if the GetFocusedElement result is not a UIElement, try getting the parent at least once before forcing things to be UIElements.